src/cohortMembers/cohortMembers.module.ts

```
@@ -65,6 +65,7 @@ import { CohortMembersCronService } from './cohortMembers-cron.service';
     PostgresModule,
     ElasticsearchModule,
     forwardRef(() => FormsModule),
+
```

ESLint

[error] Delete `⏎`

(prettier/prettier)